superset/tags/commands/create.py

```diff
@@ -95,31 +92,23 @@ def run(self) -> None:
 
     def validate(self) -> None:
         exceptions = []
-        # Validate object_id
-        if self._objects_to_tag:
-            if any(obj_id == 0 for obj_type, obj_id in self._objects_to_tag):
-                exceptions.append(TagInvalidError())
-
-            # Validate object type
-            skipped_tagged_objects: list[tuple[str, int]] = []
-            for obj_type, obj_id in self._objects_to_tag:
-                skipped_tagged_objects = []
-                object_type = to_object_type(obj_type)
-
-                if not object_type:
-                    exceptions.append(
-                        TagInvalidError(f"invalid object type {object_type}")
-                    )
-                try:
-                    model = to_object_model(object_type, obj_id)  # type: ignore
-                    security_manager.raise_for_ownership(model)
-                except SupersetSecurityException:
-                    # skip the object if the user doesn't have access
-                    skipped_tagged_objects.append((obj_type, obj_id))
-
-            self._objects_to_tag = set(self._objects_to_tag) - set(
-                skipped_tagged_objects
-            )
+        objects_to_tag = self._properties.get("objects_to_tag", [])
+        skipped_tagged_objects: list[tuple[str, int]] = []
```

Comment:
You can create these as sets, instead of converting later:

```suggestion
        objects_to_tag = set(self._properties.get("objects_to_tag", []))
        skipped_tagged_objects: set[tuple[str, int]] = set()
```